mmv1/templates/terraform/post_create/workbench_instance.go.erb

```diff
@@ -1,3 +1,13 @@
 if err := waitForWorkbenchInstanceActive(d, config, d.Timeout(schema.TimeoutCreate) - time.Minute); err != nil {
   return fmt.Errorf("Workbench instance %q did not reach ACTIVE state: %q", d.Get("name").(string), err)
 }
+
+if p, ok := d.GetOk("desired_state"); ok && p.(string) == "STOPPED" {
```

Comment:
here's the state modification post_create code

Reply:
Ah ok, I must have missed this!